dist/arch/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: Tristan Webb <olafthefrog@gmail.com>
+# template start; name=git; version=1;
+# template start; name=git-source; version=1;
+pkgname=skylobby-git
+pkgver=0.7.49.r0.g475d09b
+pkgdesc="A Spring RTS lobby and replay viewer written in Clojure"
+pkgrel=1
+arch=('any')
+makedepends+=('git' 'clojure')
+depends+=('java-runtime>=14')
+license=('MIT')
+url="https://github.com/skynet-gh/skylobby"
+source+=(
+  "${_gitname:=${pkgname%-git}}::${_giturl:-git+$url}"
+  "local://skylobby.sh"
+)
+sha256sums=('SKIP'
+            '7675e1020c41d4e53005c82a4944696e6d4a2030785c4069485ecf1f90d1ea19')
```

Comment:
Won't need integrity check after this is incorporated into source. Will get the skylobby binary wrapper from git